### ISSUE TITLE
Remove application label from library module

### ISSUE THREAD

**Issue by mseroczynski** [JustKiddingBaby/VercodeEditText#1]
```Error:Execution failed for task ':app:processXXXManifest'.
> Manifest merger failed : Attribute application@label value=(@string/label_application) from AndroidManifest.xml:16:7-48
  	is also present at [com.justkiddingbaby:vercodeedittext:1.0.3] AndroidManifest.xml:11:18-50 value=(@string/app_name).
  	Suggestion: add 'tools:replace="android:label"' to <application> element at AndroidManifest.xml:11:3-14:17 to override.```

**Comment by JustKiddingBaby** [comment]
@mseroczynski please add `tools:replace="android:label"` to <application> element at `AndroidManifest.xml`

**Comment by mseroczynski** [comment]
The problem is in your library project so please fix it in library project.

**Comment by Rainer-Lang** [comment]
Make a PR. :relaxed:

**Comment by JustKiddingBaby** [comment]
@mseroczynski the problem has been fixed,please use `v1.0.4`
